py/src/braintrust/logger.py

```diff
@@ -3416,6 +3417,71 @@ def render_message(render: Callable[[str], str], message: PromptMessage):
     return base
 
 
+def _create_custom_render():
```

Comment:
nit: a good instance where moving to a separate file makes it clear that we don't need/use the `chevron` import anywhere else. Or we could move the `import chevron` into this function.

Reply:
Hmm looks like we also use it elsewhere in prompt.build, e.g. L3571